composer.json

```diff
@@ -107,7 +107,7 @@
         "symfony/symfony": ">=3.4.26",
         "tecnickcom/tcpdf": "^6.2.12",
         "tijsverkoyen/css-to-inline-styles": "^2.2",
-        "twig/twig": "^2.6.2"
+        "twig/twig": "^1.38"
```

Comment:
this commit name is misleading: you have removed [twig 2 support](https://github.com/PrestaShop/PrestaShop/blob/develop/composer.json#L113).

How about "Removed Twig 2 support" instead?

Reply:
You're still able to change the composer.json if you want to use Twig 2. But we never say we support twig 2, and never release a version with Twig 2.

Reply:
It's not for me, its for agencies like OhmyWeb for instance: say what you want, I just try to avoid another community shitstorm.

Reply:
This is exactly what I said, if they play with the composer.json / composer.lock, they are able to change dependencies for their usage. But right now, we never said we support Twig2 and we never release Twig2 in the zip file.